contrib/elasticsearch2/values.yaml

```diff
@@ -0,0 +1,55 @@
+# Default values for elasticsearch2.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+storage:
+  accessModes: [ "ReadWriteOnce" ]
+  capacity: 1Gi
+
+image:
+  repository: elasticsearch
+  tag: 2.4.6
+  pullPolicy: IfNotPresent
+
+# Whether we should create an ingress rule for the elastichq dashboard
+ingress:
+  enabled: true
+  host: kooper.dyn.ncsa.edu
+
+# Drop-in values for elasticsearch.yml config file:
+elasticsearch:
+  config:
+    cluster.name: "clowder"
+    path.data: "/usr/share/elasticsearch/data"
```

Comment:
i would probably just make these defaults since these are where the volumes are mounted. I'd be afraid somebody changed them and we ended up not saving the data to the persistent volume.

Reply:
Good call, that's probably wise

Reply:
Moved these out of `values.yaml` and into `templates/configmap.yaml`